authentication/views.py

```diff
@@ -285,21 +288,125 @@ def post(self, request):
         send_email_notification.delay(payload)
 
         response = {
-            "data": {
-                "client_company": serializer.data,
-                "message": "Your request to create a company has been\
-                    received, please wait for approval from landville admin."
-            }
+            "client_company": serializer.data,
+            "message": "Your request to create a company has been "
+                       "received, please wait for approval from landville "
+                       "admin."
         }
 
         return Response(response, status=status.HTTP_201_CREATED)
 
-    def check_client_admin_has_company(self, id_value):
+    @staticmethod
+    def check_client_admin_has_company(id_value):
         """
         Checks if client admin admin already has a company
         """
         return bool(Client.active_objects.filter(client_admin_id=id_value))
 
+    def patch(self, request):
+        """
+        Handle Update a user's company details once created.
+        :param request:
+        :return: company Details
+        """
+        data = request.data
+        company = self.get_queryset()
+        serializer = self.serializer_class(company[0], data=data, partial=True)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+
+        response = {
+            "client_company": serializer.data,
+            "message": "Successfully Update to company."
```

Comment:
Great work @bl4ck4ndbr0wn, please check the grammar here.

Reply:
Done.